HARJA-344 Lisää primary keyt partitioille, jotta tarkastus- ja toteumataulujen DELETE:t siirtyvät

jarnovayrynen opened the pull request.

jarnovayrynen committed:
HARJA-344 Lisää primary keyt partitioille, jotta tarkastus- ja toteumataulujen DELETE:t siirtyvät

K1ll3rF0x commented on tietokanta/src/main/resources/db/migration/V1_1081__.sql:
```diff
@@ -0,0 +1,128 @@
+ALTER TABLE tarkastus_2021_q4 ADD CONSTRAINT tarkastus_2021_q4_pkey PRIMARY KEY USING INDEX tarkastus_2021_q4_id_idx;
+ALTER TABLE tarkastus_2022_q1 ADD CONSTRAINT tarkastus_2022_q1_pkey PRIMARY KEY USING INDEX tarkastus_2022_q1_id_idx;
+ALTER TABLE tarkastus_2022_q2 ADD CONSTRAINT tarkastus_2022_q2_pkey PRIMARY KEY USING INDEX tarkastus_2022_q2_id_idx;
+ALTER TABLE tarkastus_2022_q3 ADD CONSTRAINT tarkastus_2022_q3_pkey PRIMARY KEY USING INDEX tarkastus_2022_q3_id_idx;
```
Puuttuuko 2022 q4?

K1ll3rF0x replied:
Ja 2023 q1

jarnovayrynen replied:
Puuttuihan ne, mikä on outoa, kun tein homman IDEA:n taululistasta copy pastena.. Noh, lisäilen ja kävin vielä käsipelillä kaiken läpi. Kohta ei pitäis puuttua mitään.

rckopteri reviewed (approved):
Joo. se yksi indeksi vielä tuonne, niin sitte voi mergetä.

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into HARJA-344

jarnovayrynen committed:
Lisää puuttuneet tarkastustaulut

Lisää urakk, alkanut, poistettu indeksin luonti toteumapartitioin luontifunktioon

jarnovayrynen committed:
Lisää reittitoteuman HTTP-reku
